doc/source/ray-overview/examples.rst

```diff
@@ -1062,7 +1075,7 @@ Ray Examples
         The Magic of Merlin - Shopify's New ML Platform
 
     .. grid-item-card:: :bdg-success:`Tutorial`
-        :class-item: gallery-item training
```

Comment:
`Tutorial` is a misnomer for this one -- it is a talk (albeit an interesting one)

Any proposal how to handle this @matthewdeng ?

Reply:
Maybe we could add `Talk` as an example type? cc @simran-2797